src/fides/api/task/conditional_dependencies/privacy_request/privacy_request_data.py

```diff
@@ -71,6 +75,16 @@ def _extract_value_by_path(self, field_address: str) -> Any:
         parts: list[str] = field_address.split(".")[1:]
         current: Any = self.privacy_request
 
+        # Handle location convenience fields - these are derived fields, not direct attributes
+        # Check if we're accessing a location convenience field before trying to extract from privacy_request
+        location_convenience_field_names = [
+            "location_country",
+            "location_groups",
+            "location_regulations",
```

Comment:
just to tie this to the above comment - this is the other area where i think it could be helpful to be referencing some shared code, rather than redefining these literals!